### PR TITLE
Blipz blog

### PR DESCRIPTION
**Pull request by sleenen**
Hey guys, small blog post request by Bas, because we needed to do something for Blipz :)

### CODE REVIEW

**Comment by sleenen** [comment]
I wil update this PR, need to add a photo!

**Review by florian-vorenkamp — DISMISSED** [review]
Looks good to me.

File: content/blog/my-first-day-at-skyworkz.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+id: blog
+title: My first day at Skyworkz
+description: Stefan reflects on his first day at Skyworkz; visiting the AWS Summit in Amsterdam and looking ahead to Blipz Conference.
+author: Stefan Leenen
+tags:
+  - AWS Summit 2023 Amsterdam
```
Comment (bennycornelissen) [inline content/blog/my-first-day-at-skyworkz.md]: Given history I'm not expecting us to write another blog about this particular AWS Summit. That would make this particular tag not too useful. More generic tagging (e.g. `Blipz, AWS`) would be my preference.

File: content/blog/my-first-day-at-skyworkz.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+id: blog
+title: My first day at Skyworkz
```
Comment (bennycornelissen) [inline content/blog/my-first-day-at-skyworkz.md]: Only a part of your blog is about this? 😉 For a blogpost with this title I would expect a post where you tell us more about who you are and what you do, etc. But this blogpost was more about going to a conference on your first day with a segway to Blipz. Which is fine, but I think the title should reflect that. 

How about: "Joining Skyworkz and hitting a conference on my first day"

File: content/blog/my-first-day-at-skyworkz.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+* Clear Expectations: People expect their supervisor or manager to provide clear and specific expectations regarding their job performance. This includes understanding goals, objectives, key performance indicators, as well as any deadlines or targets they need to meet.
+
+While this list may sound clinical, it is essential to make someone feel welcome and ensure they have everything they need. Simple gestures like having a prepared workspace and setting up their email address can go a long way in making a positive impression.
+
+## My First Day Experience
+For most of us, the first day on a new job is likely a mix of the aforementioned expectations. Getting to know numerous new people and trying to remember their names can be overwhelming. Fortunately, I already knew a few individuals at Skyworkz due to our previous collaborations. So, what did I do when my expectations were already met before my first day? Well, I decided to attend a conference!
```
Comment (bennycornelissen) [inline content/blog/my-first-day-at-skyworkz.md]: How were all your other expectations already met? 😄

File: content/blog/my-first-day-at-skyworkz.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+The keynote talk was particularly interesting, and it came as no surprise that "Generative AI" was the buzzword of the event. AWS showcased numerous services and shared their innovative ideas. While most of these are not yet available to the general public, AWS users should definitely keep an eye on these developments.
+
+### CodeWhisperer
+One service I wanted to highlight is [Amazon CodeWhisperer](https://aws.amazon.com/codewhisperer/). It aligns with tools like ChatGPT and Github Co-pilot, focusing on code generation and working with AWS. CodeWhisperer claims to provide code suggestions specifically tailored to AWS APIs, aiming to generate better working code for any AWS services you use.
+
+Although I understand that conferences like this primarily serve as marketing tools, I wished there were more in-depth talks or workshops. Some of my new colleagues shared the same sentiment. Nonetheless, it was a well-organized event, and I had an enjoyable day overall!
+
+## Blipz On The Radar
```
Comment (bennycornelissen) [inline content/blog/my-first-day-at-skyworkz.md]: Speaking of a segway 😆

**Review by bennycornelissen — DISMISSED** [review]
Some small nitpicks from my side. I'll leave it up to you to decide whether you want to address them or just ship it.

**Review by bennycornelissen — APPROVED** [review]
LGTM! 🎉

**Pull request merged**